appengine/datastore/app.js

```diff
@@ -37,7 +37,7 @@ const datastore = Datastore();
  *
  * @param {object} visit The visit record to insert.
  */
-function insertVisit(visit) {
+async function insertVisit(visit) {
```

Comment:
Ditto to @MylesBorins above comment - do we need this `async`?

Reply:
Done. Thanks!